src/main/java/com/exasol/projectkeeper/XPathSplitter.java

```diff
@@ -0,0 +1,45 @@
+package com.exasol.projectkeeper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class splits xPath expressions at the root level slashes.
+ */
+public class XPathSplitter {
+    private XPathSplitter() {
+        // empty on purpose
+    }
+
+    /**
+     * Split the xPath expression at the root level slashes.
+     * 
+     * @param xPath xPath expression
+     * @return fragments
+     */
+    public static List<String> split(final String xPath) {
+        int nestLevel = 0;
+        final List<String> fragments = new ArrayList<>();
+        StringBuilder fragmentBuilder = new StringBuilder();
```

Comment:
Can it be final?

Reply:
no, it is overwritten for example in line 34. Sadly StringBuilder has no `clear()`